components/logstash/scripts/preconfigure_elasticsearch_properties.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from cloudify import ctx
+
+
+def main():
+    """Get elasticsearch configuration needed to generate logstash.conf."""
+    target = ctx.target
+    source = ctx.source.instance.runtime_properties
+    source['es_endpoint_ip'] = (
+        target.instance.runtime_properties['es_endpoint_ip']
+    )
+    source['es_endpoint_port'] = target.node.properties['es_endpoint_port']
+
+
+if __name__ == '__main__':
```

Comment:
Why is this needed?
I think that by default these scripts are evaluated by script plugin..